qds_sdk/cluster.py

```diff
@@ -24,7 +24,7 @@ class Cluster(Resource):
     """
 
     rest_entity_path = "clusters"
-    api_version = "v1.2"
+    api_version = "v1.3"
```

Comment:
If we are changing the default version to 1.3, then we would need to make changes at other places as well for example in `qds.py`, `qubole.py`. Also we need to update comments at various places in the code.